gym/wrappers/normalize.py

```diff
@@ -130,10 +130,9 @@ def step(self, action):
         obs, rews, terminateds, truncateds, infos = self.env.step(action)
         if not self.is_vector_env:
             rews = np.array([rews])
-        self.returns = self.returns * self.gamma + rews
-        rews = self.normalize(rews)
-        dones = np.logical_or(terminateds, truncateds)
-        self.returns[dones] = 0.0
+        dones = np.logical_or(terminateds, truncateds)            
+        self.returns = self.returns * self.gamma * (1-dones) + rews
```

Comment:
Should this not be `(1-terminated)` as in cases of truncation, the state value should not be normalized

Reply:
Oh, yes, you're right. I'll make the change, add a test case too, and then move it to gymnasium.